test/unit/file_processor_test.rb

```diff
@@ -46,20 +46,20 @@ class FileProcessorTest < Minitest::Test
     test "#call provides node processor with the correct ancestors" do
       offense = mock
       @node_processor_factory.expects(:for).returns(@node_processor)
-      @node_processor.expects(:call).with do |node, ancestors:|
+      @node_processor.expects(:call).with do |node, ancestors|
         Node.class?(node) && # class Hello; world; end
           Node.class_or_module_name(node) == "Hello" &&
           ancestors.empty?
       end
-      @node_processor.expects(:call).with do |node, ancestors:|
+      @node_processor.expects(:call).with do |node, ancestors|
         parent = ancestors.first # class Hello; world; end
         Node.constant?(node) && # Hello
           Node.constant_name(node) == "Hello" &&
           ancestors.length == 1 &&
           Node.class?(parent) &&
           Node.class_or_module_name(parent) == "Hello"
       end
-      @node_processor.expects(:call).with do |node, ancestors:|
+      @node_processor.expects(:call).with do |node, ancestors|
```

Comment:
This threw a `ArgumentError: missing keyword: :ancestors` with Ruby 3.0. I assume there's a bug with mocha and Ruby 3, but I really don't want to go down that rabbit hole right now so I just changed the method to not use keyword arguments. I don't think it significantly degrades the design.

I added a sorbet signature to make up for it 😉